### PR TITLE
Drop Node 4 support #744

### PR DESCRIPTION
**Pull request by kohgpat**
Hello. It's my attempt to fix https://github.com/styleguidist/react-styleguidist/issues/744 . Hope I've made everything right.

I've a little question about package-lock.json. After I made changes I installed dependencies again with `npm install` and got a different package-lock.json and added it to the commit. Is this correct? Just want to be sure package-lock.json behaves like yarn.lock . I'm not using npm 5 currently.

Thank you.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,12 +1,10 @@
 language: node_js
 node_js:
-  - 4
   - 6
   - 8
   - 9
 matrix:
```
Comment (sapegin) [inline .travis.yml]: We can remove the whole `matrix` block now.

**Comment by kohgpat** [comment]
Updated PR.

**Comment by sapegin** [comment]
Could you remove changes in `package-lock.json` too?

**Comment by kohgpat** [comment]
Oops, I was sure I've revert it already. Sure.

**Comment by kohgpat** [comment]
Fixed.

**Comment by kohgpat** [comment]
@sapegin Don't want to bother but is everything ok this time?

**Comment by sapegin** [comment]
YES!!! I've approved the PR and will include it in 7.0.0 release (don't know yet when though ;-)

**Pull request merged**